libp2p/src/builder/phase/build.rs

```diff
@@ -1,28 +1,34 @@
+use std::time::Duration;
+
 #[allow(unused_imports)]
 use super::*;
 
 use crate::SwarmBuilder;
-use libp2p_core::Transport;
+use libp2p_core::{transport::timeout::TransportTimeout, Transport};
 use libp2p_swarm::Swarm;
 
 pub struct BuildPhase<T, B> {
     pub(crate) behaviour: B,
     pub(crate) transport: T,
     pub(crate) swarm_config: libp2p_swarm::Config,
+    pub(crate) connection_timeout: Duration,
 }
 
-const CONNECTION_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(10);
+#[allow(unused)] // used in `../swarm.rs` but due to feature flag combinations, clippy gives an unnecessary warning.
+pub(crate) const DEFAULT_CONNECTION_TIMEOUT: Duration = Duration::from_secs(10);
```

Comment:
the old constant was used in this same file, but it isn't anymore. I think it would be clearer to move it to `../swarm.rs` where most items have similar `allow` annotations and where the new constant is actually used, thus removing the need to comment about a different file

Reply:
Sorry for the really late comment. You are totally right, it makes more sense. It's done. Thanks.